samply/src/windows/etw_gecko.rs

```diff
@@ -116,6 +116,10 @@ pub fn profile_pid_from_etl_file(context: &mut ProfileContext, etl_file: &Path)
                 // nothing, but we don't want a marker for it
             }
             "MSNT_SystemTrace/StackWalk/Stack" => {
+                if !context.is_in_time_range(timestamp_raw) {
```

Comment:
I wonder if this should use referenced_timestamp_raw instead - that's what we use for the sample timestamp.